config/policy-controller/manager.yaml

```diff
@@ -59,5 +59,5 @@ spec:
           requests:
             cpu: 10m
             memory: 64Mi
-      serviceAccountName: controller-manager
+      serviceAccountName: policy-controller
```

Comment:
You likely will end up with a conflict here when pulling these into the mgc manifests, so probably do need to rename these, but maybe for consistency we could just prepend "policy" to the usual names? "policy-controller-manager", "policy-manager-role" etc... ?